must-gather/collection-scripts/logs/gather_logs_restore

```diff
@@ -12,14 +12,14 @@ timeout=$7
 skip_tls=$8
 
 # Gather restore describe and logs
-mkdir -p "{object_collection_path}"
+mkdir -p "${object_collection_path}"
 echo "[cluster=${cluster}][ns=${ns}] Gathering 'velero restore describe ${restore}'"
 if [ "$timeout" = "0s" ]; then
-    oc -n ${ns} exec $(oc -n ${ns} get po -l component=velero -o custom-columns=name:.metadata.name --no-headers) -- /bin/bash -c "/velero describe restore ${restore} --insecure-skip-tls-verify=${skip_tls} --details" &> "${object_collection_path}/restore-describe-${restore}.txt" &
+    oc -n ${ns} exec $(oc -n ${ns} get po -l component=velero -o custom-columns=name:.metadata.name --no-headers) -- /bin/bash -c "timeout 30s /velero describe restore ${restore} --insecure-skip-tls-verify=${skip_tls} --details" &> "${object_collection_path}/restore-describe-${restore}.txt" &
```

Comment:
See https://github.com/openshift/oadp-operator/pull/816#discussion_r967486740.

request-timeout do not kill velero CLI process  since request was replied to by api server. Without `timeout` executing in velero container, you would be stuck waiting for velero CLI forever.

Reply:
ok... I'm just saying it appears to me we don't need an if/else on the full call.  Just need an if / else on the timeout setting.   Maybe I'm just reading it incorrectly :)

Reply:
ya.. pulling this out into a full file now makes this easier to see.. /shakes_fist_at_github
not a big deal either way.. you could add --request-time=${timeout} and just have one line of said command. Meh.. not a big deal :)

Reply:
oc only executed once. Easier to read logic in .sh than in bash command.